src/test/java/com/shift4/ChargesTest.java

```diff
@@ -122,4 +122,21 @@ void shouldCreateChargeOnceWhenIdempotencyKeyIsUsed() {
         assertThat(second.getId()).isEqualTo(charge.getId());
     }
 
+    @Test
+    void shouldCreateChargeUsingExplicitMerchant() {
+        // given
+        Shift4Gateway gatewayWithExplicitMerchant = createGateway("SECRET_KEY", true);
```

Comment:
maybe createExplicitMerchantGateway? true as 2nd parameter does not look self explanatory

Reply:
done